lib/moon_web/components/themes_select/rtl_switcher.ex

```diff
@@ -3,34 +3,18 @@ defmodule MoonWeb.Components.ThemesSelect.RtlSwitcher do
 
   use Moon.StatelessComponent
```

Comment:
do we still need this component at all?

Reply:
Not needed. I had deleted this component, but probably reverted the change by accident and didn't notice.